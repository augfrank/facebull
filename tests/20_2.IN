M0   C0   C1  2948
M1   C0   C3  2878
M2   C0   C7  2505
M3   C0   C8  2923
M4   C0   C11  2634
M5   C0   C14  2602
M6   C0   C18  2611
M7   C1   C2  2307
M8   C1   C3  2577
M9   C1   C8  2557
M10   C1   C9  2239
M11   C1   C18  2843
M12   C2   C4  2283
M13   C2   C6  2507
M14   C2   C11  2060
M15   C2   C14  2157
M16   C3   C0  2320
M17   C3   C1  2130
M18   C3   C8  2969
M19   C3   C9  2421
M20   C3   C11  2009
M21   C4   C2  2458
M22   C4   C6  2044
M23   C4   C8  2780
M24   C4   C14  2162
M25   C4   C15  2277
M26   C4   C17  2419
M27   C4   C19  2244
M28   C5   C0  2838
M29   C5   C2  2587
M30   C5   C8  2853
M31   C5   C14  2217
M32   C5   C16  2204
M33   C5   C17  2424
M34   C5   C19  2995
M35   C6   C3  2752
M36   C6   C9  2713
M37   C6   C13  2060
M38   C6   C17  2085
M39   C7   C3  2449
M40   C7   C4  2959
M41   C7   C5  2210
M42   C7   C11  2473
M43   C7   C18  2969
M44   C8   C2  2292
M45   C8   C7  2767
M46   C8   C9  2492
M47   C8   C10  2317
M48   C8   C15  2965
M49   C8   C16  2402
M50   C8   C17  2893
M51   C8   C18  2673
M52   C8   C19  2358
M53   C9   C5  2791
M54   C9   C10  2463
M55   C9   C17  2979
M56   C10   C0  2092
M57   C10   C4  2307
M58   C10   C12  2056
M59   C10   C15  2453
M60   C10   C16  2784
M61   C11   C4  2329
M62   C11   C10  2524
M63   C11   C13  2483
M64   C11   C17  2050
M65   C12   C1  2806
M66   C12   C2  2172
M67   C12   C4  2670
M68   C13   C4  2464
M69   C13   C8  2792
M70   C13   C10  2521
M71   C13   C17  2751
M72   C14   C1  2151
M73   C14   C2  2050
M74   C14   C3  2034
M75   C14   C9  2592
M76   C14   C10  2592
M77   C14   C15  2187
M78   C14   C18  2917
M79   C15   C9  2609
M80   C15   C10  2242
M81   C15   C11  2423
M82   C15   C18  2105
M83   C15   C19  2364
M84   C16   C0  2756
M85   C16   C2  2346
M86   C16   C4  2972
M87   C16   C10  2605
M88   C16   C11  2080
M89   C16   C14  2098
M90   C17   C5  2794
M91   C18   C6  2245
M92   C18   C8  2655
M93   C18   C14  2240
M94   C18   C15  2411
M95   C18   C19  2991
M96   C19   C0  2143
M97   C19   C6  2923
M98   C19   C12  2909
M99   C19   C13  2918
