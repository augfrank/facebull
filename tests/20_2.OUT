45453
2 10 12 20 25 32 37 39 52 54 58 64 66 69 72 82 89 90 91 96
